package.json

```diff
@@ -1,6 +1,6 @@
 {
-  "name": "react-media-query-hoc",
-  "version": "1.0.3",
+  "name": "react-media-query-hoc-with-context",
```

Comment:
I’m sorry, we had to release this fix on our npm to avoid serious performance issues. We submitted this by mistake.

 I’ll remove this from the merge request

Reply:
@jooj123 : Fixed